pre_commit_mirror_maker/main.py

```diff
@@ -70,16 +71,30 @@ def main(argv: Optional[Sequence[str]] = None) -> int:
     )
     args = parser.parse_args(argv)
 
+    minimum_pre_commit_version = '0'
+
+    if args.types_or:
+        match_key = 'types_or'
+        match_val = f'[{args.types_or}]'
```

Comment:
unlike `types`, `types_or` is usually multiple things

Reply:
How would you like to enter it? I assumed `--types-or=a,b,c` similar to `--types=text,py`. An argparse list?

Reply:
action='append' probably